docs/acme-divergences.md

```diff
@@ -11,7 +11,7 @@ that simply fetch a resource (certificate, order, authorization, or challenge).
 
 ## [Section 6.6](https://tools.ietf.org/html/rfc8555#section-6.6)
 
-For all rate-limits, Boulder includes a `Link` header to additional documentation on rate-limiting. Only rate-limits on `duplicate certificates` and `certificates per registered domain` are accompanied by a `Retry-After` header.
+For all rate-limits, Boulder includes a `Link` header to additional documentation on rate-limiting.
```

Comment:
Does this count as a divergence? Nothing in RFC8555 says we shouldn't do this...